lib/server.js

```diff
@@ -8,6 +8,24 @@ const SimpleAuthPolicy = require('./simple-auth-policy');
 
 const HANDSHAKE_TIMEOUT = 3000;
 
+const _initApplications = function(applications) {
```

Comment:
This function doesn't need to start with an underscore, it isn't exported anyway.